web_tooltip_field/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2023 Komit - Cuong Nguyen Mtm
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Web Tooltip Field",
+    "version": "14.0.1.0.0",
+    "summary": "Showing a Field ToolTip in Odoo",
+    "category": "Tools",
+    "website": "https://github.com/OCA/web",
+    "author": "KOMIT, Odoo Community Association (OCA)",
+    "maintainers": ["cuongnmtm"],
+    "license": "AGPL-3",
+    "depends": ["base", "web"],
+    "data": [
+        "security/ir.model.access.csv",
+        "views/ir_model_fields_help_tooltip_view.xml",
+    ],
+    "application": True,
```

Comment:
I'm not sure we can considre this module as an application, an application in odoo world is mainly a new entry in the main menu I think.

```suggestion
```